### PR TITLE
Eslint globals and some code cleanup

### PR DESCRIPTION
**Pull request by josemigallas**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -11,6 +11,11 @@
     "jest": true,
     "jasmine": true
   },
+  "globals": {
+    "fixture": true,
+    "$": true,
```
Comment (didierofrivia) [inline .eslintrc]: we are aiming to not having jQuery as a global, at least not in our modules

File: spec/javascripts/Stats/stats_response_codes.spec.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ describe('StatsResponseCodeSource', () => {
 
 describe('StatsResponseCodeChart', () => {
   beforeEach(() => {
-    fixture.set(`<div id="chart"></div><div id="no-data">No data 'ere mate!</div>`)
+    fixture.set('<div id="chart"></div><div id="no-data">No data \'ere mate!</div>')
```
Comment (didierofrivia) [inline spec/javascripts/Stats/stats_response_codes.spec.js]: It's ok to use template strings

Comment (josemigallas) [reply spec/javascripts/Stats/stats_response_codes.spec.js]: But it really isn't:
screen shot 2018-12-19 at 09 17 15
It's not [standard.js](https://standardjs.com/rules.html#rules) compliant (second rule).

File: .eslintrc (hunk 1 above)
Comment (josemigallas) [reply .eslintrc]: OK, this is wrong then.

File: spec/javascripts/Stats/stats_response_codes.spec.js (hunk 2 above)
Comment (didierofrivia) [reply spec/javascripts/Stats/stats_response_codes.spec.js]: Ok, so we are allowing only back ticks on substitution.

**Pull request merged**